source/ways-of-working/environments/external-access.html.md.erb

```diff
@@ -0,0 +1,102 @@
+---
+title: External Access in Demo
+last_reviewed_on: 2022-03-30
+review_in: 6 months
+weight: 10
+---
+
+# <%= current_page.data.title %>
+
+The demo environment is used to demonstrate new features to external stakeholders. 
+
+For guidance on how to deploy an application to the demo environment, see [Common Pipeline](https://hmcts.github.io/ways-of-working/common-pipeline/common-pipeline.html#building-infrastructure-in-the-demo-environment).
```

Comment:
this is probably a better link for application deployment: https://hmcts.github.io/ways-of-working/new-component/gitops-flux.html#application-config-in-flux. the existing link only mentions infra.

Possibly the existing section should just have a link to config added so that you can cover infra and application in one link?

Reply:
Haven't updated this link yet, can you clarify what you mean here? 

> Possibly the existing section should just have a link to config added so that you can cover infra and application in one link?

Reply:
I mean if you follow the link you added it just talks about infrastructure not about deploying the application.

Also you should avoid absolute links as when developing locally it will take you to the production site not the local one